systems/px4_default_imu_sensor.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+<system name="PX4 IMU Sensor">
+<!--  =================================================================
+This is a generic JSBSim system file to simulate the IMU behavior for PX4.
+  =================================================================  -->
+
+    <!-- Enable Individual Component Failure -->
+    <property value="1">px4/acc_x/on</property>
+    <property value="1">px4/acc_y/on</property>
+    <property value="1">px4/acc_z/on</property>
+
+    <property value="1">px4/gyro_x/on</property>
+    <property value="1">px4/gyro_y/on</property>
+    <property value="1">px4/gyro_z/on</property>
+
+    <channel name="Accelerometer-X" execute="px4/acc_x/on">
+        <accelerometer name="Default-ACC-X">
+            <location unit="M">
+                <x> 0 </x>
+                <y> 0 </y>
+                <z> 0.345 </z>
+            </location>
```

Comment:
Is there a way to avoid duplicating this for the other axes?

Reply:
I believe this is a constraint of the JSBSim accelerometer function. There are a few functions that allow XML variables to be passed so that you could accomplish something like this where the property is declared once: 

```XML
                <x> <property> px4/imu/loc_x </property>  </x>
                <y> <property> px4/imu/loc_y </property>  </y>
                <z> <property> px4/imu/loc_z </property>  </z>
```
I dug into the backend of JSBSim (and tried a few combination of the above) and unfortunately the only input it accepts are [numeric](https://github.com/JSBSim-Team/jsbsim/blob/b42f4238880ae1729acd55894381f74a0fe1ca13/src/models/flight_control/FGAccelerometer.cpp#L61-L63).

Reply:
Ok, let's keep it as is then.